src/graph/classdef-graph.hh

```diff
@@ -26,6 +26,7 @@
 
 #include "graph.hh"
 #include "../hb-ot-layout-common.hh"
+#include <cstdint>
```

Comment:
Curious why you added this.

Reply:
I think a previous iteration of my code for this used that, but it's not needed anymore. I'll remove.